packages/flutter_tools/dart_test.yaml

```diff
@@ -1,5 +1,9 @@
-tags:
-  "no_coverage":
-  "create":
-  "integration":
- 
+# package:test configuration
```

Comment:
Does removing the tags mean we can no longer tell build_runner to build _only_ those tags?

Is ther esome other way we can avoid building the world for each build_runner run?  It takes a much longer time to build the whole flutter_tools test package than building a subshard.

Reply:
FWIW, this appears to currently be broken, but when it was first sharded it seemed to work.

Reply:
If build_runner is building things it wasn't told to run, that seems like a bug in build_runner?

Reply:
This has been fixed in `build_runner`. Theoretically it should be a matter of enabling it. We already use it in the engine web tests. Works pretty well there.

See also:

- https://github.com/flutter/flutter/issues/37810
- https://github.com/dart-lang/build/pull/2415